app/helpers/tags_helper.rb

```diff
@@ -11,4 +11,21 @@ def taggable_path(taggable_type, tag_name)
     end
   end
 
+  def nav_element(match_controller, path = root_path, legend = '', options = {}, &block)
```

Comment:
Bueno, supongo que es un tema de dónde poner la línea. A mí este helper se me parece bastante  los que te da rails, con opciones para configurar diferentes cosas. Por eso a mí no me canta tanto. Me parece que aunque sea una solución relativamente grande para un caso puntual, se compensa siguiendo cosas que hace rails y por lo tanto respetando el principio de la mínima sorpresa.

Dicho esto a mí ambas opciones me parecen válidas con sus pros y sus contras. Puesto que las consecuencias de la decisión no me las como yo, sino que os las coméis los que estáis en el día a día del proyecto, lo suyo es que llegués a un consenso ;-)